archinstall/lib/user_interaction/disk_conf.py

```diff
@@ -73,19 +73,15 @@ def select_disk(dict_o_disks: Dict[str, BlockDevice]) -> BlockDevice:
 	"""
 	drives = sorted(list(dict_o_disks.keys()))
 	if len(drives) >= 1:
-		for index, drive in enumerate(drives):
-			print(
-				f"{index}: {drive} ({dict_o_disks[drive]['size'], dict_o_disks[drive].device, dict_o_disks[drive]['label']})"
-			)
+		title = str(_('You can skip selecting a drive and partitioning and use whatever drive-setup is mounted at /mnt (experimental)')) + '\n'
+		title += str(_('Select one of the disks or skip and use "/mnt" as default"'))
```

Comment:
Double quotes are unnecessary here, it breaks translation string
```suggestion
		title += str(_('Select one of the disks or skip and use /mnt as default'))
```

Reply:
Indeed, thanks